python/lsst/qserv/tests/benchmark.py

```diff
@@ -177,6 +174,13 @@ def _parseFile(self, qF, withQserv):
             elif withQserv and line.startswith("-- withQserv"):
                 # strip the "-- withQserv" text
                 qText.append(line[13:])
+            elif line.endswith("-- noQserv"):
+                if withQserv:
+                    # skip this line
+                    pass
+                else:
+                    # strip the "-- noQserv" text
+                    qText.append(line[:-10])
```

Comment:
I think the reason why "-- noQserv" is at the end of the line and "-- withQserv" at the beginning is that you can feed this file directly to mysql without any replacements, i.e. it looks like correct mysql query (but not qserv query).